internal/restconfig/restconfig.go

```diff
@@ -256,47 +319,95 @@ func (rcp *Producer) RunOnSelectedPrefixedContext(prefix string, function PerCon
 	return nil
 }
 
-func (rcp *Producer) AddKubeConfigFlag(cmd *cobra.Command) {
+// RunOnAllContexts runs the given function on all accessible non-prefixed contexts.
+func (rcp *Producer) RunOnAllContexts(function PerContextFn, status reporter.Interface) error {
+	if rcp.inCluster {
+		return runInCluster(function, status)
+	}
+
+	if rcp.defaultClientConfig.overrides.CurrentContext != "" {
+		// The user has explicitly chosen a context, use that only
+		return rcp.RunOnSelectedContext(function, status)
+	}
+
+	if rcp.defaultClientConfig != nil {
```

Comment:
To reduce indentation:

```suggestion
	if rcp.defaultClientConfig == nil {
	    return nil
	}
```